### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/Slider/SliderItem/SliderItem.module.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@import 'styles/fontMixin';
+@import 'styles/colors';
+@import 'styles/positionMixin';
+
+.wrapper {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    
+    .picture {
```
Comment (valentinPortside) [inline src/components/Slider/SliderItem/SliderItem.module.scss]: .picture img {
   ...styles
}

File: src/components/Slider/SliderItem/SliderItem.module.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        
+        .desk {
+            @include fontStyle(400, 24px, 29px);
+            margin-bottom: 4px;
+        }
+    }
+}
+
+@media screen and (max-width: 975px) {
+    .wrapper {
```
Comment (valentinPortside) [inline src/components/Slider/SliderItem/SliderItem.module.scss]: .wrapper .text {
   ...styles
}

fix everywhere pls

File: src/components/Slider/Slider.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import { SliderItem } from './SliderItem'
+
+import styles from './Slider.module.scss'
+
+export const Slider = () => {
+    const [numberPic, setNumberPic] = useState(0)
+
+    const refContainer = useRef<HTMLDivElement>(null) 
+
+    const sliderPics = [
```
Comment (valentinPortside) [inline src/components/Slider/Slider.tsx]: move it to out of the Component

File: src/components/Slider/SliderItem/SliderItem.module.scss (hunk 1 above)
Comment (Ksenaaa) [reply src/components/Slider/SliderItem/SliderItem.module.scss]: fixed

File: src/components/Slider/SliderItem/SliderItem.module.scss (hunk 2 above)
Comment (Ksenaaa) [reply src/components/Slider/SliderItem/SliderItem.module.scss]: fixed

File: src/components/Slider/Slider.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

**Pull request merged**